Signal/src/view controllers/CallViewController.swift

```diff
@@ -519,7 +519,6 @@ class CallViewController: UIViewController, CallDelegate {
 
     func updateCallUI(callState: CallState) {
         let textForState = localizedTextForCallState(callState)
-        Logger.info("\(TAG) new call status: \(callState) aka \"\(textForState)\"")
```

Comment:
I'm sorry it looks like you just took out the call state logging without replacing it with something else.  Do you want to remove it?